src/api/.keystone/config.js

```diff
@@ -261,6 +319,39 @@ var extendGraphqlSchema = (schema) => (0, import_schema.mergeSchemas)({
           return { success: false };
         }
       }
+    },
+    Query: {
+      getCompaniesHouseInformation: async (root, variables) => {
+        try {
+          const { companiesHouseNumber } = variables;
+          console.info("Calling Companies House API for ", companiesHouseNumber);
+          const sanitisedRegNo = companiesHouseNumber.padStart(8, "0");
+          const response = await (0, import_axios.default)({
+            method: "get",
+            url: `${companiesHouseURL}/company/${sanitisedRegNo}`,
+            auth: { username, password: "" },
```

Comment:
Should the password be blank?

Reply:
yes, same as dtfs, only key is required which is the username